README_de-DE.md

```diff
@@ -2,19 +2,19 @@
 
 [![Sag danke!](https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg)](https://saythanks.io/to/RomuloOliveira)
 
-Ein Leitfaden um die Wichtigkeit von Commit-Messages zu verstehen und diese richtig zu formulieren.
+Ein Leitfaden, um die Wichtigkeit von Commit-Messages zu verstehen und diese richtig zu formulieren.
 
-Verstehe was ein Commit ist, warum es wichtig ist eine gute Commit-Message zu schreiben, Best-Practices und ein paar Tipps um eine saubere Commit-Historie zu planen oder vielleicht sogar nachträglich noch anzupassen.
+Verstehe was ein Commit ist, warum es wichtig ist, eine gute Commit-Message zu schreiben, Best-Practices und ein paar Tipps, um eine saubere Commit-Historie zu planen oder vielleicht sogar nachträglich noch anzupassen.
```

Comment:
maybe 'Bewährte Praktiken' instead of 'Best-Practices'

Reply:
maybe `gute Praxis`?

Reply:
Yes, also fine; much better as the words in English